src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,14 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+};
+
+export const TodoList:React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo todo={todo} />
+    ))}
```

Comment:
don't forget to add keys by mapping the list of elements
```suggestion
    {todos.map(todo => (
      <TodoInfo key={todo.id} todo={todo} />
    ))}
```